pkg/api/monitoring/metrics.go

```diff
@@ -110,13 +131,16 @@ type MetricsI interface {
 	RecordEgressRequestDur(target string, elapsed float64)
 	RecordIngressRequestCnt(code, method, url string)
 	RecordIngressRequestDur(code, method, url string, elapsed float64)
+	RecordDatabaseStatusErrorCnt(target string)
+	RecordSqsStatusErrorCnt(target string)
 	Use(e *echo.Echo)
 }
 
 // Metrics contains the metrics gathered by the instance and its path
 type Metrics struct {
-	ingressReqCnt, egressReqCnt *prometheus.CounterVec
-	ingressReqDur, egressReqDur *prometheus.HistogramVec
+	ingressReqCnt, egressReqCnt           *prometheus.CounterVec
+	ingressReqDur, egressReqDur           *prometheus.HistogramVec
+	databaseStatusErrCnt, sqsStatusErrCnt *prometheus.CounterVec
```

Comment:
We should't have specifics in this module. Instead we can record errors in general as `errCnt`